subsys/bluetooth/mesh/Kconfig

```diff
@@ -292,6 +299,21 @@ config BT_MESH_TX_SEG_MAX
 	  which leaves 56 bytes for application layer data using a
 	  4-byte MIC and 52 bytes using an 8-byte MIC.
 
+config BT_MESH_TX_SEG_RETRANS_CNT
+	int "Number of retransmit attempts"
```

Comment:
This ends up in the same menu as the rest of the mesh configuration, could you specify that this is for segmented transport messages? E.g. "Transport message segment retransmit attempts"